### ISSUE TITLE
Is this behavior intentional?

### ISSUE THREAD

**Issue by TonyShor23** [ValveSoftware/Dota2-Gameplay#19568]
### Ability name

Doom, and Unfettered 

### Description

HP just melts, when Lifestealer with Sange and Yasha uses Ufettered and gets Doomed, his HP just melts, -1000 HP in a matter of seconds. I thought Abilities deal damage over seconds, you get status resistance = you get less seconds of debuffs, and with it, less damage, instead, you get all the damage faster. Is this intentional? 

### Example Match ID (and possibly Timestamp)

Demo Hero

**Comment by Dum0sss** [comment]
Yes, it is. Spells has fixed damage, when its a dot, like doom, it deals it by ticks. But if you speed up ticks the damage per second increases. Status resistance means that it decreases the effects duration on you if its possible (eul's and other things cant be reduced.), damage doesnt get reduced with status resistance.

**Comment by TonyShor23** [comment]
> Yes, it is. Spells has fixed damage, when its a dot, like doom, it deals it by ticks. But if you speed up ticks the damage per second increases. Status resistance means that it decreases the effects duration on you if its possible (eul's and other things cant be reduced.), damage doesnt get reduced with status resistance.

Fine. I'll allow it.
